### PR TITLE
Manually Process Host Application On Error

### PR DESCRIPTION
**Pull request by SudharakaP**
Resolve https://github.com/opencollective/opencollective/issues/6408

[test-osc.webm](https://user-images.githubusercontent.com/12435965/221607743-19ab79f5-7d64-41ae-81a6-6cd19d83940a.webm)

### CODE REVIEW

File: test/server/graphql/v2/mutation/CreateCollectiveMutations.test.js [hunk 1]
```diff
@@ -139,160 +119,4 @@ describe('server/graphql/v2/mutation/CreateCollectiveMutations', () => {
       expect(memberInvitationActivities[1].data.memberCollective.name).to.eq('Another admin');
     });
   });
-
-  describe('with GitHub repository', async () => {
```
Comment (SudharakaP) [inline test/server/graphql/v2/mutation/CreateCollectiveMutations.test.js:143]: @znarf : Removing these as we are just showing error and letting the the application process so no failure to test.

Comment (znarf) [reply test/server/graphql/v2/mutation/CreateCollectiveMutations.test.js:143]: Ok, let's review the implications of this quietly before merging.

Comment (SudharakaP) [reply test/server/graphql/v2/mutation/CreateCollectiveMutations.test.js:143]: @znarf : What do you mean by "review the implications quietly"? Do I need to do anything on my end? 🙂

**Comment by znarf** [comment]
After "reviewing the implications quietly", I came to the conclusion that it was better solved by https://github.com/opencollective/opencollective-api/pull/8619

**Pull request closed without merging**